db/migrate/20160513024658_udate_vendors_status.rb

```diff
@@ -0,0 +1,8 @@
+class UdateVendorsStatus < ActiveRecord::Migration
```

Comment:
Just be careful to rollback and recreate instead of changing this file directly.